source/libs/utils.ts

```diff
@@ -30,6 +30,15 @@ export const getRepoBranch = (): string | false => {
 	return false;
 };
 
+export const getCurrentBranch = (): string | undefined => {
+	const recentCommitsLink = select('link[title^="Recent Commits to"]')!;
+	// Will be something like https://github.com/sindresorhus/refined-github/commits/master.atom
+	const href = recentCommitsLink.getAttribute('href')!;
+	const group = /\/commits\/(.+)\.atom$/.exec(href);
+
+	return group ? group[1] : undefined;
```

Comment:
```ts
return select('[aria-current="page"]')!
	.pathname // <user>/<repo>/tree/<branch/slashed-branch>
	.split('/')
	.slice(4) // Drops everything and including `tree`
	.join('/');
```

Reply:
LGTM, it seems to be more resilient than my approach.

Reply:
The only problem I can think of is that when we are in the repo root (`https://github.com/sindresorhus/refined-github`), the above code will yield an empty string. It might works just fine with this feature since we have another check about whether we are in repo root, but might not be appropriate for a more general utility function. Should we fallback to default branch in this case? Is it possible?

Reply:
Let's revert to `select('.branch-select-menu .css-truncate-target').textContent`, it works all the time.

Reply:
I'm still getting `branch/still-b…` instead of `branch/still-branch` by using that [here](https://github.com/simonkotwicz/refined-github/blob/branch/still-branch/.travis.yml#L6). I might have did something wrong?

Reply:
Indeed, they must have changed it. This seems to work:

```js
select('.branch-select-menu .css-truncate').title
```

Reply:
Like I stated in the PR description, I think this would only work for branch but not for tag? (I don’t have my laptop to verify now)